### PR TITLE
feat: CIP-1694

### PR DESCRIPTION
**Pull request by vladimirvolek**

### CODE REVIEW

File: src/definitions.yaml [hunk 1]
```diff
@@ -312,9 +317,45 @@ paths:
...
+
+  /governance/dreps/{drep_id}/votes:
+    $ref: ./paths/api/governance/dreps/{drep_id}/votes.yaml
+
+  /governance/proposals:
+    $ref: ./paths/api/governance/proposals/index.yaml
+
+  /governance/proposals/{tx_hash}/{cert_index}:
+    $ref: ./paths/api/governance/proposals/{tx_hash}/{cert_index}/index.yaml
+
```
Comment (sorki) [inline src/definitions.yaml:350]: Missing `/governance/proposals/{tx_hash}/{cert_index}/parameters` 
(`./paths/api/governance/proposals/{tx_hash}/{cert_index}/parameters.yaml`)

**Comment by sorki** [comment]
Epoch params and proposal params both missing `pvt_p_p_security_group`

**Review by sorki — CHANGES_REQUESTED** [review]

**Comment by sorki** [comment]
Another new protocol param missing - `min_fee_ref_script_cost_per_byte`

File: src/definitions.yaml (hunk 1 above)
Comment (mmahut) [reply src/definitions.yaml:366]: Yes, that is a proposal that contains new protocol parameters.

**Review by slowbackspace — APPROVED** [review]

**Pull request merged**